feature: Se implemento las pruebas y clase Verdadero & Falso Con Pena…

LuisParedes1 opened the pull request:
Pruebas de Verdadero & Falso con Penalidad. Comentarios cortos sobre la implementación

[ComentariosNuevos.pdf](https://github.com/gylopezgiles/TP2/files/4974732/ComentariosNuevos.pdf)

LuisParedes1 committed:
feature: Se implemento las pruebas y clase Verdadero & Falso Con Penalidad

gylopezgiles commented on src/main/java/edu/fiuba/algo3/modelo/pregunta/VerdaderoFalsoClasico.java:
```diff
@@ -33,6 +33,16 @@ public int establecerPuntuacion(List<Opcion> opciones) {
+                .findAny();
+
+        return opcion.get();
+    }
```
este metodo esta bien en cuanto a codigo. Pero veo 2 cosas aca, 1) no tiene mucho sentido agregarlo solamente para usarlo en las pruebas, basicamente le estas agregando a todas las preguntas una responsabilidad mas solo para usarlo en las pruebas, eso no esta del todo bien. 2) el findany lo que hace es que si encuentra un elemento de la lista que cumple la condicion de filtrado te lo devuelve, pero no sabes si hay mas de 1, por lo tanto si vos necesitas que sea 1 elemento unico no te sirve para eso, y si tenes mas de 1 te puede devolver cualquiera

LuisParedes1 replied:
Ok es cierto. Voy a borrar el metodo porque rompe con el principio de unica responsabilidad

LuisParedes1 committed:
Fix: Se elimino obtenerOpcionCorrecta porque rompia con el principio de unica responsabilidad

Feature: Se agregaron los diagramas de secuencia para:
	* alResponderIncorrectamenteDebeRestarUnPunto
	* alResponderCorrectamenteDebeSumarUnPunto

	Ademas se implemento el diagrama de secuencia de creacion general de una pregunta de verdadero y falso en plantuml.
